lib/js/atoms/Input/Input.vue

```diff
@@ -0,0 +1,250 @@
+<template>
+	<div
+		class="a-input"
+		:class="{ '-success': success, '-error': error || !!errorMessage, '-disabled': disabled }"
+	>
+		<v-text-field
+			:id="id"
+			:autofocus="autofocus"
+			:disabled="disabled"
+			:error-messages="errorMessage"
+			:error="error"
+			:hint="helpMessage"
+			:persistent-hint="!!helpMessage"
+			:readonly="readonly"
+			:success="success"
+			:value="value"
+			filled
+		>
+			<template #prepend-inner>
+				<template v-if="leftIcon">
+					<icon :icon="leftIcon" :size="ICON_SIZES.X_SMALL"></icon>
+				</template>
+			</template>
+
+			<template #append>
+				<template v-if="rightIconComputed">
+					<icon :icon="rightIconComputed" :size="ICON_SIZES.X_SMALL"></icon>
+				</template>
+			</template>
+
+			<template #label>
+				<div>
+					<span class="a-input__label">
+						{{ label }}
+					</span>
+					<span v-if="labelAddition" class="a-input__labelAddition">
+						{{ labelAddition }}
+					</span>
+				</div>
+			</template>
+
+			<template #message>
+				<div class="a-input__helpMessage">
+					<icon
+						v-if="error || errorMessage"
+						class="a-input__labelErrorIcon"
+						:icon="ICONS.FA_EXCLAMATION_TRIANGLE"
+						:size="ICON_SIZES.XX_SMALL"
+					></icon>
+					<span>
+						{{ errorMessage || helpMessage }}
+					</span>
+				</div>
+			</template>
+		</v-text-field>
+	</div>
+</template>
+
+<style lang="scss" scoped>
+@import '../../../styles/settings/colors';
+@import '../../../styles/settings/typography';
+
+.a-input {
+	&__labelAddition {
+		color: $color-primary;
+	}
+
+	&.-error {
+		.a-input__labelAddition {
+			color: $color-danger;
+		}
+
+		.a-input__helpMessage {
+			color: $color-danger;
+		}
+	}
+
+	&.-success {
+		.a-input__labelAddition {
+			color: $color-success;
+		}
+
+		.a-input__helpMessage {
+			color: $color-success;
+		}
+	}
+
+	&.-disabled {
+		.a-input__labelAddition {
+			color: rgba($color-total-black, 0.38);
+		}
+	}
+}
+</style>
+
+<style lang="scss">
+@import '../../../styles/settings/colors';
+@import '../../../styles/settings/spacings';
+
+.a-input {
+	.v-input__slot {
+		margin-bottom: $space-xxxs !important;
+		background: rgba($color-firefly-black, 0.06) !important;
+	}
+
+	&__labelErrorIcon > svg {
+		margin-right: $space-xxxxs;
+	}
+
+	&__helpMessage {
+		display: flex;
+		align-items: center;
+	}
+
+	.v-input--is-focused .v-input__slot {
+		background: rgba($color-firefly-black, 0.12) !important;
+	}
+
+	&.-error .v-input__slot {
+		background: $color-danger-background !important;
+	}
+
+	&.-success .v-input__slot {
+		background: $color-success-background !important;
+	}
+
+	.v-input--is-disabled .v-input__slot::before {
+		border-image: none !important;
+	}
+
+	&.-success {
+		.v-input__prepend-inner {
+			fill: $color-success;
```

Comment:
using `fill` will break duotone FA icons. We have to use `color` instead.

Reply:
Most likely you'll need to setup vue-svg-loader in storybook and library build to make it working